test_steps/base_methods.py

```diff
@@ -24,5 +29,22 @@ def get_text(self, by, locator) -> str:
         return self.find_elem(by, locator).text
 
     def wait_for_elem_visible(self, by, locator, seconds):
-        return WebDriverWait(self.browser, int(seconds)).until(
+        return WebDriverWait(self.browser, seconds).until(
             EC.visibility_of_element_located((by, locator)))
+
+    def goto_url(self, url):
+        self.browser.get(DOMAIN + url)
+
+    def clear_input(self, by, locator):
+        self.find_elem(by, locator).clear()
+
+    def get_elem_value(self, by, locator) -> str:
+        return self.browser.find_element(by, locator).get_attribute('value')
+
+    def scroll_to_elem(self, by, locator):
+        target = self.find_elem(by, locator)
```

Comment:
I was thinking location_once_scrolled_into_view... but this may work... though I haven't tried :)